src/features/hotspots/setup/HotspotSetupBluetoothInfoScreen.tsx

```diff
@@ -26,10 +26,14 @@ const HotspotSetupBluetoothInfoScreen = () => {
   const { showOKCancelAlert } = useAlert()
 
   const subtitle1 = t(
-    `hotspot_setup.pair.${hotspotType === 'RAK' ? 'rak_' : ''}subtitle_1`,
+    `hotspot_setup.pair.${hotspotType === 'RAK' ? 'rak_' : ''}${
```

Comment:
It would be good to simplify this logic, as more manufactures add hotspots it is going to get a little crazy.

Would something like this work? 
`const subtitle1 = t(`hotspot_setup.pair.${hotspotType.toLowerCase()}_subtitle_1`)`

Reply:
Would be great to use something like that for these strings and the ones in `HotspotSetupPowerScreen`